tests/mocha/event_procedure_change_return_test.js

```diff
@@ -11,6 +11,9 @@ import {sharedTestSetup, sharedTestTeardown} from './test_helpers/setup_teardown
 
 
 suite('Procedure Change Return Event', function() {
+  const DEFAULT_TYPES = null;
+  const NON_DEFAULT_TYPES = [];
```

Comment:
Not a blocker, but: I'm not sure I love this naming scheme.  I find it somehow confusing—certainly, I was a little confused when I first saw this in a previous PR review.  I think it might be better to use names that make it slightly more clear that these are arbitrary test values.  Maybe just `TEST_TYPES_1` `TEST_TYPES_2` or something?

(Also applies to e.g. `DEFAULT_NAME` / `NON_DEFAULT_NAME` in `event_procedure_rename_test.js` below.)

Reply:
These are not arbitrary test values :/ the `DEFAULT` values match the default values defined in the classes. E.g. the value for `getReturnTypes` is `null`. 

I wanted to make sure that when we call `setReturnTypes` in the `running` test suite, we were actually changing it to be different than the default, i.e. actually triggering a change.

I thought giving them these names would help with readability, but if it's not helpful, we can try something else.

I'm gonna go ahead and merge this, but happy to refactor in a future PR =)